packages/docs/src/pages/components/forms.mdx

```diff
@@ -26,13 +26,13 @@ import {
   onSubmit={e => e.preventDefault()}>
   <Label htmlFor='username'>Username</Label>
   <Input
-    name='username'
+    id='username'
```

Comment:
Thanks! These elements should probably have both `name` and `id` attributes, shouldn't they?

Reply:
As far as I know `name` can be skipped for labels access. If you want, I can add it back :)

Reply:
@jxnblk added back. Let me know if anything else you think we need to change.